src/settings.rs

```diff
@@ -153,6 +153,8 @@ pub struct SerialSettingsPlatform<C, const Y: usize> {
     /// The storage mechanism used to persist settings to between boots.
     pub storage: Flash,
 
+    pub active_settings: C,
```

Comment:
Meh. There are at least four places where we keep the setting now...

I'd rather have the poll call into serial settings return some sort of flag that the app then uses to push the settings. Or save that flag in the platform.